internal/backends/mcu/README.md

```diff
@@ -93,3 +99,11 @@ Then you can simply run with `cargo run`
 ```sh
 CARGO_TARGET_THUMBV6M_NONE_EABI_LINKER="flip-link" CARGO_TARGET_THUMBV6M_NONE_EABI_RUNNER="probe-run --chip RP2040" cargo +nightly run -p printerdemo_mcu --features=mcu-pico-st7789 --target=thumbv6m-none-eabi --release
 ```
+
+### STM32H735G-DK
+
+Using [probe-run](https://github.com/knurling-rs/probe-run) (`cargo install probe-run`)
+
+```sh
+CARGO_TARGET_THUMBV7EM_NONE_EABIHF_RUNNER="probe-run --chip STM32H735IGKx" cargo +nightly run -p printerdemo_mcu --features=i-slint-backend-mcu/stm32h735g,i-slint-backend-mcu/panic-probe --target=thumbv7em-none-eabihf --release
```

Comment:
I get an compile error with with the panic probe, so suggesting this:

```suggestion
CARGO_TARGET_THUMBV7EM_NONE_EABIHF_RUNNER="probe-run --chip STM32H735IGKx" cargo +nightly run -p printerdemo_mcu --features=i-slint-backend-mcu/stm32h735g --target=thumbv7em-none-eabihf --release
```

This is the error:

```sh
error: duplicate lang item in crate `panic_semihosting` (which `ft5336` depends on): `panic_impl`.
  |
  = note: the lang item is first defined in crate `panic_probe` (which `i_slint_backend_mcu` depends on)
  = note: first definition in `panic_probe` loaded from /Users/simon/src/slint-mcu/target/thumbv7em-none-eabihf/release/deps/libpanic_probe-c0aa84673dc4a15c.rmeta
  = note: second definition in `panic_semihosting` loaded from /Users/simon/src/slint-mcu/target/thumbv7em-none-eabihf/release/deps/libpanic_semihosting-80a962ef50aab939.rmeta

error: could not compile `i-slint-backend-mcu` due to previous error
```